fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -938,6 +938,38 @@ private void analyzeAddPartition(OlapTable olapTable, List<PartitionDesc> partit
         }
     }
 
+    private int calAvgBucketNumOfRecentPartitions(OlapTable olapTable, int recentPartitionNum) {
+        int avgBucketNum = 0;
+        if (olapTable.getPartitions().size() < 5) {
+            int backendNum = GlobalStateMgr.getCurrentSystemInfo().getBackendIds().size();
+            // When POC, the backends is not greater than three most of the time.
+            // The bucketNum will be given a small multiplier factor for small backends.
+            if (backendNum <= recentPartitionNum) {
+                avgBucketNum = 2 * backendNum;
+            } else if (backendNum <= 6) {
```

Comment:
1.5x and  1.2x may cause very unbalanced load between BEs when PoC testing.

Reply:
The PoC will only have the same backends. So I think the bucket is enough. And also the PoC backend will not be installed many nodes